runtime/near-vm-logic/src/types.rs

```diff
@@ -35,3 +38,6 @@ pub enum PromiseResult {
     Successful(Vec<u8>),
     Failed,
 }
+
+/// Profile of gas consumption, +1 for Wasm bytecode execution cost.
+pub type ProfileData = Rc<RefCell<[u64; Actions::count() + ExtCosts::count() + 1]>>;
```

Comment:
Can we hide it behind the feature as well? It seems like this introduces `RefCell` and `Rc` towards the `vm-logic` from `std`, so this creates the dependencies.

Reply:
As this type is used in signatures, it will clutter all code using it with the feature. Original code didn't use `RefCell` / `Rc` but had explicit lifetimes, and Max requested to rewrite it. What would we consider least evil?